glpk/meta.yaml

```diff
@@ -17,6 +17,16 @@ requirements:
   build:
     - gmp      [not win]
 
+test:
```

Comment:
Need to add Cython as a test-time dependency.  

This is a nice approach to hide compilation complexity, by the way.  A good example for other projects that have this sort of thing to test.